optimum/utils/logging.py

```diff
@@ -262,3 +267,8 @@ def reset_format() -> None:
 
     for handler in handlers:
         handler.setFormatter(None)
+
+
+@lru_cache(None)
```

Comment:
```suggestion
@lru_cache
```
is enough no?

Reply:
No, the default is 128: https://docs.python.org/3/library/functools.html#functools.lru_cache